src/fprime/fbuild/cmake.py

```diff
@@ -280,6 +280,8 @@ def generate_build(
             ),
             args.keys(),
         )
+        # Creating a file to mark the directory as a F Prime directory
+        open(build_dir / ".fprime-build-dir", "w")
```

Comment:
We should either close the file, or better imo is to use pathlib with for example 

```suggestion
        Path(build_dir / ".fprime-build-dir").touch()
```